src/javascript/binary/base/page.js

```diff
@@ -854,6 +854,20 @@ Contents.prototype = {
             $('.unbind_later').off();
         }
     },
+    has_mlt_mf_enabled: function() {
```

Comment:
@jy-binary it was a poor choice of naming on my part. It's not actually checking broker codes. I have renamed the function and variables.